rotkehlchen/exchanges/utils.py

```diff
@@ -118,3 +119,46 @@ def query_binance_exchange_pairs(location: Location) -> dict[str, BinancePair]:
         database_pairs = gdb_binance.get_all_binance_pairs(location)
         pairs = {pair.symbol: pair for pair in database_pairs}
     return pairs
+
+
+def pair_symbol_to_base_quote(
+        symbol: str,
+        asset_deserialize_fn: Callable[[str], AssetWithOracles],
+        five_letter_assets: tuple[str, ...],
+) -> tuple[AssetWithOracles, AssetWithOracles]:
+    """Turns a symbol product into a base/quote asset tuple.
```

Comment:
does this work fine for gemni and bybit? And only used there? Then mention it

Reply:
Yes works for both and is only used in this two places